ccp/modules/datashield-setup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+if [ "$ENABLE_DATASHIELD" == true ]; then
+  log INFO "DataSHIELD setup detected -- will start DataSHIELD services."
+  OVERRIDE+=" -f ./$PROJECT/modules/datashield-compose.yml"
+  OPAL_DB_PASSWORD="$(echo \"This is a salt string to generate one consistent password for Opal. It is not required to be secret.\" | openssl rsautl -sign -inkey /etc/bridgehead/pki/${SITE_ID}.priv.pem | base64 | head -c 30)"
+  DATASHIELD_CONNECT_SECRET="$(echo \"This is a salt string to generate one consistent password as the DataShield Connect secret. It is not required to be secret.\" | openssl rsautl -sign -inkey /etc/bridgehead/pki/${SITE_ID}.priv.pem | base64 | head -c 30)"
+  if [ ! -e /tmp/bridgehead/opal-cert.pem ]; then
+    mkdir -p /tmp/bridgehead/
```

Comment:
in my case starting the bridgehead failed here, as /tmp/bridgehead was created with root permissions and not owned by the bridgehead user
Same issue for /var/cache/bridgehead

Reply:
I did a lot of permission stuff now can you try again?

Reply:
@Threated : Is it possible to implement a solution for this? Or can it only be done manually?

Reply:
It should work automatically now is what I was trying to say

Reply:
Unfortunately, I still have problems with that.